pkg/datagatherer/gke/gke.go

```diff
@@ -18,6 +18,15 @@ type GKEDataGatherer struct {
 	credentialsPath string
 }
 
+type GKEDataGathererConfig struct {
+	Project     string `mapstructure:"project"`
+	Location    string `mapstructure:"location"`
+	Zone        string `mapstructure:"location"` // Deprecated
+	Cluster     string `mapstructure:"cluster"`
+	Credentials string `mapstructure:"credentials"`
+	DataPath    string `mapstructure:"data-path"`
```

Comment:
See this comment about  `mapstructure`: https://github.com/jetstack/preflight/pull/82#discussion_r377681362